string arm_name
TransformVerbose[] transforms
ArmJointState[] joint_states
